specification/compute/resource-manager/readme.md

```diff
@@ -209,52 +209,52 @@ directive:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
 
 ```
-### Tag: package-2020-06-30
+### Tag: package-2020-09-30
 
-These settings apply only when `--tag=package-2020-06-30` is specified on the command line.
+These settings apply only when `--tag=package-2020-09-30` is specified on the command line.
 
-``` yaml $(tag) == 'package-2020-06-30'
+``` yaml $(tag) == 'package-2020-09-30'
 input-file:
 - Microsoft.Compute/stable/2020-06-01/compute.json
 - Microsoft.Compute/stable/2020-06-01/runCommands.json
 - Microsoft.Compute/stable/2019-04-01/skus.json
 - Microsoft.Compute/stable/2020-06-30/disk.json
-- Microsoft.Compute/stable/2019-12-01/gallery.json
+- Microsoft.Compute/preview/2020-09-30/gallery.json
+- Microsoft.Compute/preview/2020-09-30/sharedGallery.json
 - Microsoft.ContainerService/stable/2017-01-31/containerService.json
 ```
 
-### Tag: package-2020-06-30-only
+### Tag: package-2020-09-30-only
 
-These settings apply only when `--tag=package-2020-06-30-only` is specified on the command line.
+These settings apply only when `--tag=package-2020-09-30-only` is specified on the command line.
 
-``` yaml $(tag) == 'package-2020-06-30-only'
+``` yaml $(tag) == 'package-2020-09-30-only'
 input-file:
-- Microsoft.Compute/stable/2020-06-30/disk.json
+- Microsoft.Compute/preview/2020-09-30/gallery.json
+- Microsoft.Compute/preview/2020-09-30/sharedGallery.json
 ```
 
-### Tag: package-2020-09-30
+### Tag: package-2020-06-30
```

Comment:
pls add new section for 2020-09-30 and 2020-09-30-only instead of updating 06-30

Reply:
I swapped the position of 2020-09-30 and 2020-06-30 since I updated the latest version to 2020-09-30. The 2020-06-30 and 2020-09-30 are always there.